README.md

```diff
@@ -0,0 +1,39 @@
+# ngrok agent docker image
+
+## Introduction
+
+This repository contains code related to building and releasing the [ngrok docker image](https://hub.docker.com/r/ngrok/ngrok)
```

Comment:
Total nit, but I personally prefer [reference style](https://github.github.com/gfm/#reference-link) links for docs like this. That would be:

```
[ngrok docker image][ngrok-dockerhub]
```

and then at the bottom of the document, the [list reference definition](https://github.github.com/gfm/#link-reference-definition)

```
[ngrok-dockerhub]: https://hub.docker.com/r/ngrok/ngrok
```

Same nit applies to all the links, but I won't block on it.

Reply:
did not know that was a thing! fixing...